apps/frontend/src/libs/modules/pwa-notification/pwa-notification.module.ts

```diff
@@ -0,0 +1,36 @@
+import { GRANTED_PERMISSION } from "./libs/constants/constants.js";
+
+class PWANotification {
```

Comment:
```suggestion
class PushNotification {
```
change it everywhere